app/src/main/java/com/weatherxm/ui/devicedetails/DeviceDetailsActivity.kt

```diff
@@ -250,12 +254,41 @@ class DeviceDetailsActivity : BaseActivity() {
             device.profile,
             device.isActive,
         )
+        if (!model.device.isOnline()) {
+            binding.status.setOnClickListener {
+                navigator.showDeviceAlerts(this, model.device)
+            }
+        } else {
+            binding.status.setOnClickListener(null)
+            binding.status.isClickable = false
+        }
 
         if (device.address.isNullOrEmpty()) {
             model.fetchAddressFromCell()
         } else {
             binding.address.text = device.address
         }
+
+        setAlerts(device.alerts)
+    }
+
+    private fun setAlerts(alerts: List<DeviceAlert>) {
+        val alertsWithoutOffline = alerts.dropWhile {
+            it.alert == DeviceAlertType.OFFLINE
+        }
+        if (alertsWithoutOffline.size > 1) {
+            binding.alertChip.text = getString(R.string.issues, alertsWithoutOffline.size)
+            binding.alertChip.setVisible(true)
+        } else if (alertsWithoutOffline.size == 1) {
+            if (alertsWithoutOffline[0].alert == DeviceAlertType.NEEDS_UPDATE) {
+                binding.alertChip.text = getString(R.string.update_required)
+            } else if (alertsWithoutOffline[0].alert == DeviceAlertType.LOW_BATTERY) {
+                binding.alertChip.text = getString(R.string.low_battery)
+            }
+            binding.alertChip.setVisible(true)
+        } else {
+            binding.alertChip.setVisible(false)
+        }
```

Comment:
Minor, maybe you can use something like this under the `if`s to make things more clear
```
binding.alertChip.setVisible(!alertsWithoutOffline.isEmpty())
```

Reply:
Good catch Panteli, you are right! I just fixed it.